xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml

```diff
@@ -44,14 +44,13 @@
     #set ($name = 'templateContent')
     #set ($editedDocument = $xwiki.getDocument("$stringtool.removeEnd($className, 'Class')Template"))
     ## Don't load the WYSIWYG editor when editing the class, because it's too heavy.
-    #set ($useWysiwygEditor = false)
   #else
     ## We are editing an application entry so the content must be read from / written to the current document.
     #set ($name = 'content')
     #set ($editedDocument = $tdoc)
     ## Use the preferred content editor.
```

Comment:
The comment needs to be moved also (or removed if you think the code is obvious).

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3594/commits/4371e521d0183839094f7947c7ca8ef58c649620 :+1: 
I moved the comment around. It doesn't hurt to keep it here as long as it's valid :)